### PR TITLE
Akshola00/issue31

### LINKED ISSUE #31: [FE]: Create Store Form (Step 3 of the Stepper)
# [FE]: Create Store Form (Step 3 of the Stepper)  

> [!IMPORTANT]  
Before applying, please read the [Contributors Guide](https://github.com/StarShopCr/contributors-guide). Following the guide increases your chances of being assigned to this task.  

## Description  
Implement the third step of the store registration using the design provided in the Figma file. This step is part of a multi-step form (Stepper) and should strictly adhere to the specified layout and style.  

## Requirements  
1. **File to Edit**:  
   - Modify the file located at `src/components/register/steps/ThreeStepContent.tsx`.  

2. **Design**:  
   - Follow the Figma design exactly as provided to ensure consistency with the overall UI [Figma Design Link](https://www.figma.com/design/OsBwtlrTh74tPs3S3MZm1N/StarShop-Figma?node-id=0-1&node-type=canvas&t=pVTHQecm3MDfgGXN-0), you have to use the `Store Form` Section.  
   - Use proper styling techniques to maintain responsiveness and visual appeal.  

###  ⚠️ You don't have to create this, because it's already created
Captura de pantalla 2024-12-09 a las 1 09 44 p  m

### You just have to create the form 
Captura de pantalla 2024-12-09 a las 1 27 07 p  m

### PR DESCRIPTION
**Pull request by Akshola00**
# 🚀 StarShop Pull Request

- [] Closes #31
- [] Added tests (if necessary)
- [] Run tests
- [x] Run formatting
- [] Evidence attached
- [] Commented the code

---

## 📝 Changes description
made neccasry changes to the stepthree content file to make sure the design looks pixel perfect 
used shadcn UI for select box

---

## 📸 Evidence (A photo is required as evidence)
image

---

## ⏰ Time spent breakdown
1 day to create the whole form

---

## 🌌 Comments

---

Thank you for contributing to StarShop, we are glad that you have chosen us as your project of choice and we hope that you continue to contribute to this great project, so that together we can make our mark at the top!

### CODE REVIEW

**Comment by Akshola00** [comment]
@KevinLatino pls review

File: src/components/register/steps/StepThreeContent.tsx [hunk 1]
```diff
@@ -1,9 +1,100 @@
...
-    )
-}
\ No newline at end of file
+  return (
+    <>
+      <div className="flex flex-col items-center mt-2">
+        <UploadPhotoProfile />
+        <div className="flex flex-col items-center justify-center w-[428px] px-14 pt-8 pb-14 bg-[#0F102C] rounded-[3rem] shadow-lg border-4 border-[#3D3C75]">
+          <div className="w-full space-y-4">
+            <div className="flex flex-col w-full mt-0">
```
Comment (KevinLatino) [inline src/components/register/steps/StepThreeContent.tsx:23]:
```suggestion
            <div className="flex flex-col w-full mt-[4rem]">
```

add mt pls

**Review by KevinLatino — COMMENTED** [review]
Please, create the inputs and form as in the Step One Content, here is the link of the PR: 

https://github.com/StarShopCr/StarShop-Frontend/pull/37

**Comment by Akshola00** [comment]
suggested hanges implemented, pls review

**Pull request merged**